packages/runtime/test/transport/files.test.ts

```diff
@@ -95,6 +96,14 @@ describe("File upload", () => {
         expect(response).toBeSuccessful();
     });
 
+    test("can upload a file with undefined as expiry date and automatically set expiry date to maximum value", async () => {
```

Comment:
Cumbersome description. How about "uploading a file without expiry date will use the default"?